BloomFilter.md

```diff
@@ -132,10 +132,11 @@ to default row group size. It is also recommended to expose the ability for sett
 parameters to end users.
 
 #### File Format
-The Bloom filter data of a column chunk, which contains the size of the filter in bytes, the
-algorithm, the hash function and the Bloom filter bitset, is stored near the footer. The Bloom
-filter data offset is stored in column chunk metadata. Here are Bloom filter definitions in
-thrift:
+Each multi-block Bloom filter is required to work for only one column chunk. The data of a multi-block Bloom
```

Comment:
Nit: Paragraphs should be separated from previous content by a blank line.

Reply:
Thanks, will update in next commit.

Reply:
Looks like you missed some of the missing blank lines.